terraform/stacks/umccr_data_portal/pipeline/main.tf

```diff
@@ -224,4 +224,25 @@ resource "aws_ssm_parameter" "sqs_somalier_extract_queue_arn" {
   type  = "String"
   value = aws_sqs_queue.somalier_extract_queue.arn
   tags  = merge(local.default_tags)
-}
\ No newline at end of file
+}
+
+# TODO: There's no clear guidance/documentation about where to plug new "apps"
+# in the (data-portal pipeline?): it's either in here or in app/ dir?
+#
+# This needs to be documented!
```

Comment:
Yup. Need to note better. Since Dracarys integration follows WES event orchestration pattern, please use pipeline here, pls.